src/server_manager/web_app/karma.conf.js

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 const {makeConfig} = require('../base.webpack.js');
+process.env.CHROMIUM_BIN = require('puppeteer').executablePath();
```

Comment:
Is this needed? Won't Karma use the default that Puppeteer returns already?

Reply:
Without this it fails to launch Chromium on my machine.  Karma looks at system default locations if you don't set that env var: https://github.com/karma-runner/karma-chrome-launcher/blob/master/index.js#L297